OpenRA.Mods.Common/Traits/Player/PlayerStatistics.cs

```diff
@@ -60,6 +57,10 @@ public int Experience
 		public int BuildingsDead;
 
 		public int ArmyValue;
+
+		// High resolution (every second) record of earnings, limited to the last minute
+		readonly Queue<int> earnedSeconds = new Queue<int>(60);
```

Comment:
This belongs up with the income-related fields 20 lines above.

Reply:
This is not a public field. So I sorted it further below. (Didn't we use to have a style rule for that?)

Reply:
Neither are `lastIncome` or `lastIncomeTick`. Maybe they can move here instead.

Reply:
Ah derp, I forgot to fix that.